en/deploy/README.md

```diff
@@ -30,7 +30,7 @@ $ git config --global user.name "Your Name"
 $ git config --global user.email you@example.com
 ```
 
-Initializing the git repository is something we need to do only once per project (and you won't have to re-enter the username and email ever again).
+Initializing the git repository is something we need to do only once per project. (And you won't have to re-enter the username and email ever again.)
```

Comment:
Starting a sentence with `and` doesn't sound right. In fact, the whole sentence could use some improvement to indicate it's global but a distinct separate step from `init`. How about `The config is done only once on your computer. Even when you start another git project on the same computer, you won't have to re-enter the username and email again.`

Reply:
I like ekohl's suggestion here.